contracts/starknet/Interfaces/ISpaceAccount.cairo

```diff
@@ -0,0 +1,113 @@
+%lang starknet
+
+from openzeppelin.account.library import AccountCallArray
+from contracts.starknet.lib.general_address import Address
+from contracts.starknet.lib.proposal_info import ProposalInfo
+from starkware.cairo.common.uint256 import Uint256
+
+@contract_interface
+namespace ISpaceaccount {
+    func getPublicKey() -> (publicKey: felt) {
+    }
+
+    func supportsInterface(interfaceId: felt) -> (success: felt) {
+    }
+
+    func setPublicKey(newPublicKey: felt) {
+    }
+
+    func isValidSignature(hash: felt, signature_len: felt, signature: felt*) -> (isValid: felt) {
```

Comment:
Really sucks to be using `camelCase` and then `snake_case` for `signature_len` and `signature` but I guess there's no way around it. Feels terribly wrong tho hahaha

Reply:
agreed